internal/zone/map_events.go

```diff
@@ -27,6 +27,10 @@ type playerRunsEvent struct {
 	nc     *structs.NcActMoveRunCmd
 }
 
+type playerEmotedEvent struct {
+	handle uint16
```

Comment:
as I mentioned, this would need to have another field which is the emoticon id: 

```suggestion
	handle uint16
	emoteID uint16
```